server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -231,6 +232,7 @@ export function configureWebSocketServices(
 	verifyMaxMessageSize?: boolean,
 	socketTracker?: core.IWebSocketTracker,
 	revokedTokenChecker?: core.IRevokedTokenChecker,
+	eventEmitter?: EventEmitter,
```

Comment:
Adding a typed event emitter here will also help explain what this param should do.  Also consider a more revealing name for it (besides eventEmitter).